### DISCUSSION TITLE
Allow preparation # auto-increment to be reset for each Collection Object

### DISCUSSION THREAD

**Discussion by grantfitzsimmons** [specify/specify7#5749] [Feature Requests]
> Our requirement for both the insect collection and the herbarium will be to have any auto-incrementing reset at the collection object level. It would be good if the formatter section had a tick-box option to allow for this to occur.
>
> This is an addition to our previous request for preparation numbering improvements ([Preparation numbering](https://app.asana.com/0/1201972579963460/1203584420764663)) which has resulted in two feature request GH issues ([Issue #2712](https://github.com/specify/specify7/issues/2712) and [Issue #2111](https://github.com/specify/specify7/issues/2111)).
>
>![image](https://user-images.githubusercontent.com/37256050/222215940-596460b2-963d-435d-800e-3e591079a7a1.png)

**Requested By:** Corinna P and Rob VH at CSIRO ([Asana](https://app.asana.com/0/1201972579963460/1204084646880692/f))

**Comment by grantfitzsimmons** [comment]
Mentioned in today's meeting with CSIRO

**Comment by maxpatiiuk** [comment]
Seems like this should be part of https://github.com/specify/specify7/issues/2111 - thus @grantfitzsimmons consider merging these issues

#2111 mentions catalog number, but many of the improvements should be applied to formatted fields in general, not just catalog numbers
